script.js

```diff
@@ -0,0 +1,83 @@
+const rangeInput = document.querySelectorAll(".range-input input"),
```

Comment:
You could add some newlines here between groups of variables and after methods to improve readability. They look cluttered and it's tougher to understand where a new method starts